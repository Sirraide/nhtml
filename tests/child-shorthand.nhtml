div > div > div {
    div (foobar)
}