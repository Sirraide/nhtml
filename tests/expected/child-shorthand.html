<div>
    <div>
        <div>
            <div>foobar</div>
        </div>
    </div>
</div>
